server/tests/common/replicated_loglet.rs

```diff
@@ -89,7 +89,6 @@ pub struct TestEnv {
     pub loglet: Arc<dyn Loglet>,
     pub metadata_writer: MetadataWriter,
     pub metadata_store_client: MetadataStoreClient,
-    pub cluster: StartedCluster,
```

Comment:
I removed passing the cluster to the test routine as it is easy to accidentally drop it, and kill the cluster in the process. We can reintroduce it as a reference if it's needed in the future.